### DISCUSSION TITLE
posts/2.-%EB%A6%AC%EC%8A%B5%EC%9C%BC%EB%A1%9C-%EC%BA%90%EB%93%9C%EB%AA%85%EB%A0%B9%EC%96%B4-%EC%82%AC%EC%9A%A9-%ED%95%B4-%EB%B3%B4%EA%B8%B0-(command)/

### DISCUSSION THREAD

**Discussion by giscus** [IsaacGoCoder/IsaacGoCoder.github.io#1] [General]
# posts/2.-%EB%A6%AC%EC%8A%B5%EC%9C%BC%EB%A1%9C-%EC%BA%90%EB%93%9C%EB%AA%85%EB%A0%B9%EC%96%B4-%EC%82%AC%EC%9A%A9-%ED%95%B4-%EB%B3%B4%EA%B8%B0-(command)/

목표 캐드에서 자체적으로 제공하는 명령어(Line, Rectangle 등)을 리습으로 실행 해 봅시다.

http://localhost:4000/posts/2.-%EB%A6%AC%EC%8A%B5%EC%9C%BC%EB%A1%9C-%EC%BA%90%EB%93%9C%EB%AA%85%EB%A0%B9%EC%96%B4-%EC%82%AC%EC%9A%A9-%ED%95%B4-%EB%B3%B4%EA%B8%B0-(command)/

**Comment by yeonmech** [comment]
잘 읽었습니다 감사합니다
